Automattic/simplenote-ios
# Render Checklists in the Markdown Preview

During testing of #258, users noticed that the markdown preview was not converting the checklist markdown syntax into a checkbox.

This PR fixes that by updating the markdown parser to `Hoextdown` (we used `Hoedown` before), which can parse the checklist syntax into HTML:

![simulator screen shot - iphone x - 2019-01-04 at 15 14 24](https://user-images.githubusercontent.com/789137/50717673-df7be900-103d-11e9-9c30-8f37bb7da3e1.png)

Unfortunately there's no pod for `Hoextdown`, so I included it in the source of the project (which is the majority of the new code in this PR).

**To Test**
* Create a checklist in the editor.
* Enable markdown for the note from the info panel.
* Swipe to view the markdown preview. The checkboxes should display!
* The checkboxes should _not_ be tappable in the preview.
